backend/controllers/projectCtrls.js

```diff
@@ -1,7 +1,69 @@
 const db = require("../models");
 
 // TODO ROUTE FUNCTIONS (NOT ACTIONS) GO HERE
+const getProject = (req, res) => {
+    db.Project.find({})
+    .then((foundProject) => {
+        if(!foundPeople) {
+            res.status(404).json({message: "Cannont find project."})
+        } else {
+            res.status(200).json({data: foundProject})
+        }
+    });
+};
+
+
+const createProject = (req, res) => {
+    req.body.password = bcrypt.hashSync(
```

Comment:
There shouldn't be a password being sent here. Grab the user ID from the session and add it to req.body as user and delete this encryption block